client/errors.go

```diff
@@ -49,8 +50,17 @@ func (e objectNotFoundError) Error() string {
 }
 
 // NewVersionError returns an error if the APIVersion required
-// if less than the current supported version
-func (cli *Client) NewVersionError(APIrequired, feature string) error {
+// if less than the current supported version.
+//
+// It performs API-version negotiation if the Client is configured
+// with this option, otherwise it it uses client's default version.
```

Comment:
```suggestion
// is less than the current supported version.
//
// It performs API-version negotiation if the Client is configured
// with this option, otherwise it uses client's default version.
```

Reply:
DOH! Thanks.

Also: I need to check why we don't have the `dupwords` linter enabled 🤔

Reply:
Fixed, and slightly rephrased PTAL